metals/src/main/scala/scala/meta/internal/metals/debug/SetBreakpointsRequestHandler.scala

```diff
@@ -19,9 +21,24 @@ private[debug] final class SetBreakpointsRequestHandler(
   def apply(
       request: SetBreakpointsArguments
   ): Future[SetBreakpointsResponse] = {
-    val path = adapters.adaptPath(request.getSource.getPath)
-    val input = path.toAbsolutePath.toInput
-    val occurrences = Mtags.allToplevels(input).occurrences
+    val path = adapters.adaptPath(request.getSource.getPath).toAbsolutePath
+
+    val topLevels = Mtags.allToplevels(path.toInput)
+    val occurrences = path.toLanguage match {
+      case Language.JAVA =>
+        def isTypeSymbol(symbol: SymbolInformation): Boolean = {
+          val kind = symbol.kind
+          kind.isClass || kind.isInterface
```

Comment:
What about enums?

Reply:
Good catch, thanks!
It turns out enums have a `class` kind, so they are already covered. Nevertheless, I have added a test case for them.